GDJS/Runtime/CustomRuntimeObjectInstanceContainer.ts

```diff
@@ -264,8 +258,10 @@ namespace gdjs {
     /**
      * Get the renderer associated to the RuntimeScene.
      */
-    getRenderer(): gdjs.CustomObjectRenderer {
-      return this._renderer;
+    getRenderer():
```

Comment:
Nice, it did not have sense to have the _renderer in this class right?

Reply:
Yes, as there is not really something to factorize between 2D and 3D renderer I though it would be clearer than an empty abstract renderer.